### ISSUE TITLE
Customize Bottom Controls

### ISSUE THREAD

**Issue by berkeugur** [hnvn/flutter_image_cropper#355]
we cant customize bottom controls. icon colors, icons, scale and rotation bar background color and  more...

**Comment by anupgupta-arg** [comment]
Yes, customizing is required like the Action on done button or cancel button

**Comment by owenmoon1** [comment]
Any update on this?
